recipe/meta.yaml

```diff
@@ -29,7 +32,10 @@ requirements:
 test:
   commands:
     - "${PREFIX}/standalone_conda/conda.exe -V"  # [unix]
+    - "${PREFIX}/standalone_conda/conda.exe create -y -p ./env_test conda" # [unix]
```

Comment:
please add a small package to make sure it's really working.  zlib is probably good.